tests/database/test_model.py

```diff
@@ -38,13 +46,19 @@ async def test_query_obj_async(self, db_async_obj):
         result = await DBSession.execute(select(self.klass).options(selectinload("*")))
         obj = result.scalar_one()
         for key, value in self.attrs.items():
-            objvalue = getattr(obj, key)
-            if isinstance(objvalue, (int, str)):
-                assert objvalue == value
+            if key == "api_key":
+                assert validate_api_key(value, obj.api_key)
+            else:
+                objvalue = getattr(obj, key)
+                if isinstance(objvalue, (int, str)):
+                    assert objvalue == value
         for key, value in self._db_obj_get_dependencies().items():
-            objvalue = getattr(obj, key)
-            if isinstance(objvalue, (int, str)):
-                assert objvalue == value
+            if key == "api_key":
+                assert validate_api_key(value, obj.api_key)
+            else:
+                objvalue = getattr(obj, key)
+                if isinstance(objvalue, (int, str)):
+                    assert objvalue == value
```

Comment:
Similar things as with `BaseTestController`, let's not special case in the base test class.

Reply:
Same thing as above, with `ModelTestBase.no_validate_attrs`.